pctasks/dataset/tests/test_dataset.py

```diff
@@ -111,7 +111,7 @@ def test_process_items_is_update_workflow(tmp_path, has_args, extra_uri) -> None
     workflow = create_process_items_workflow(
         ds_config,
         collection_config,
-        chunkset_id="${{ args.since }}",
```

Comment:
It's interesting that I used this dynamic value in the chunkset ID for the unit test, but didn't foresee this issue for the actual pipelines.